### PR TITLE
increase paranoia

### PR DESCRIPTION
**Pull request by SamStudio8**
appease friend computer

### CODE REVIEW

**Comment by zuzak** [comment]
Requesting new Travis check

**Comment by zuzak** [comment]
can you rebase this / merge with current head?

**Comment by zuzak** [comment]
I prefer the original wording.

File: modules/concern.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = {
 
 						bot.shout( to, nick + ': you seem ' + x.adj + x.hwinner + ' (score: ' + x.emotional + ')' );
 					} else if ( verbose ) { // danger phrase
-						bot.shout( to, nick + ': that is a worrying thing to say' );
+						bot.shout( to, nick + ': that is a treasonous thing to say' );
```
Comment (danhedron) [inline modules/concern.js]: This is worrying

Comment (danhedron) [inline modules/concern.js]: test

**Pull request closed without merging**